hilfe/handbuch/datenexport/index.md

```diff
@@ -0,0 +1,98 @@
+- {%icon "exclamation-triangle"%} Sind **Warnungen**, die Sie darauf aufmerksam machen, dass der Lohnbericht so nicht von ihrer Lohnbuchhaltung zu verwenden ist. Diese Punkte sollten Sie möglichst beheben.
+- {%icon "hexagon-exclamation"%} Sind Fehlermeldungen, die den Export des Lohnberichts verhindern. Bis diese behoben sind, ist ein Export nicht möglich. 
+
+{% figure caption="Beim Export dieses Lohnberichts werden wir darauf aufmerksam gemacht, dass es Probleme geben kann, wenn für Mitarbeiter Personalnummern fehlen." %}
+<img src="fehlermeldung.webp" />
+{% endfigure %}
+
+> {% icon "info-circle" %} **Hinweis:** Mit der Ausnahme von **LohnAG** erfordert der korrekte Export in Buchhaltungsprogramme immer die Erstellung von **Lohnartensätzen**. Mehr Informationen zur Konfiguration von Lohnarten finden Sie unter [Einstellungen/Buchhaltung](/hilfe/handbuch/einstellungen/buchhaltung#lohnarten).
+
+
+## Arbeitszeiten Exportieren
+
+Pentacode bietet Ihnen einige Optionen, die Arbeitszeiten einzelner Mitarbeiter oder ganzer Abteilungen zu exportieren, sei es für Archivierungszwecke, zur Überprüfung durch den Mitarbeiter oder für die Übermittlung an die Lohnbuchhaltung.
+
+### Nachweis erstellen
+
+Um den Arbeitszeitennachweis zu erstellen navigieren Sie in den Reiter **Datenexport** und klicken auf {% button label="Arbeitszeit-Nachweis" %}. Wählen Sie nun im Kalender erst einmal den Zeitraum aus, für den Sie den Nachweis erstellen wollen. Soll es ein **Monatsnachweis** sein, können Sie auch gleich einen ganzen Monat unter dem Kalender auswählen. 
+Wählen Sie nun unter **Mitarbeiter** aus, für wen Sie den Nachweis erstellen wollen. Sie können dabei alle Mitarbeiter, bestimmte Anstellungsverhältnisse, ganze Abteilungen oder einzelne Mitarbeiter auswählen. Wenn Sie **eine Abteilung und einen Mitarbeiter auswählen, werden nur die Arbeitszeiten des Mitarbeiters in dieser Abteilung exportiert!**
+
+{% figure caption="Wir werden im Arbeitszeitnachweis nur die Schichten dieses Mitarbeiters in der Abteilung **Service** sehen." %}
+<img src="mitarbeiter_filter.webp" />
+{% endfigure %}
+
+**Im Dokumentenarchiv ablegen** sichert eine Kopie des Arbeitszeitennachweises im Dokumentenarchiv des Mitarbeiters und gewährt diesem auch Zugriff darauf. So, dass mit dem Anwählen von **Unterschrift anfordern** der Mitarbeiter den Nachweis auch direkt bestätigen kann, und Ihnen die Absicherung gibt - ganz ohne das Dokument jemals audrucken zu müssen. 
+
+> Wir empfehlen Ihnen ihre Mitarbeiter stets dazu aufzufordern den Arbeitszeitennachweis mit einer Unterschrift zu bestätigen - dies gibt Ihnen Sicherheit im Falle einer Prüfung und deckt Missverständnisse frühzeitig auf. Nutzen Sie dazu wie oben beschrieben die Funktion **Unterschrift anfordern**. Davor müssen Sie jedoch dem Mitarbeiter Zugriff auf sein Dokumentenarchiv gewähren - [hier](/hilfe/handbuch/mitarbeiter/zugaenge-rechte/#berechtigungen-übersicht) können Sie diese Einstellung treffen. 
```

Comment:
Kleine Korrektur: Die Berechtigung "Dokumentenarchiv" in der MA-App ist **nicht** erforderlich. Solange der MA einen Zugang zur App hat, kann er auch Formale ausfüllen und unterschreiben. Zugriff zum Formular hat er direkt über die Startseite in der App.